src/classes/Callable_API_TEST.cls

```diff
@@ -39,6 +39,7 @@ private with sharing class Callable_API_TEST {
 
     private static final String TDTM_TRIGGER_ACTION = 'TDTM.DisableAllTriggers';
     private static final String CRLP_IS_ENABLED_ACTION = 'CRLP.iscrlpenabled';
+    private static final String RD2_IS_ENABLED_ACTION = 'settings.enableenhancedrecurringdonations';
```

Comment:
`RD2_IS_ENABLED_ACTION` -> `RD2_ENABLE_ACTION` since the action is enabling RD2 and not checking if it is enabled.